spec/lang/step/terminators.md

```diff
@@ -85,14 +85,17 @@ fn check_abi_compatibility(
         (Type::Ptr(_), Type::Ptr(_)) =>
             // The kind of pointer and pointee details do not matter for ABI.
             true,
-        (Type::Tuple { fields: caller_fields, size: caller_size, align: caller_align },
-         Type::Tuple { fields: callee_fields, size: callee_size, align: callee_align }) =>
+        (Type::Tuple { sized_fields: caller_fields, sized_head_layout: caller_head_layout, unsized_field: None },
+         Type::Tuple { sized_fields: callee_fields, sized_head_layout: callee_head_layout, unsized_field: None }) => {
```

Comment:
This means we entirely reject unsized tuples here, right?
That's fine but the comment on the `_ =>` arm needs to be updated.